Checkout/Source/Network/Models/SecurityCode/SecurityCodeResponse.swift

```diff
@@ -0,0 +1,14 @@
+// 
+//  SecurityCodeResponse.swift
+//  
+//
+//  Created by Okhan Okbay on 05/10/2023.
+//
+
+import Foundation
+
+struct SecurityCodeResponse: Decodable {
```

Comment:
Is there a reason to have SecurityCodeResponse and SecurityCodeTokenDetails which are similar data models?

Reply:
`SecurityCodeResponse` is an internal `Decodable` model which we can change anytime by our needs. `SecurityCodeTokenDetails` is a public frozen model we expose to merchants

Reply:
Not a solid reason since they are the exact same models property-wise. But can be good to keep them separate. A bit or premature optimisation maybe but it doesn't hurt that much to keep them I believe 🙂

Reply:
resolving this. Feel free to re-open if doesn't feel like been justified. @deepesh-vasthimal-cko